lib/internal/constants.js

```diff
@@ -11,13 +11,34 @@ module.exports = {
   CHAR_DOT: 46, /* . */
   CHAR_FORWARD_SLASH: 47, /* / */
   CHAR_BACKWARD_SLASH: 92, /* \ */
+  CHAR_VERTICAL_LINE: 124, /* | */
   CHAR_COLON: 58, /* : */
   CHAR_QUESTION_MARK: 63, /* ? */
   CHAR_UNDERSCORE: 95, /* _ */
   CHAR_LINE_FEED: 10, /* \n */
   CHAR_CARRIAGE_RETURN: 13, /* \r */
+  CHAR_TAB: 9, /* \t */
+  CHAR_FORM_FEED: 12, /* \f */
   CHAR_EXCLAMATION_MARK: 33, /* ! */
   CHAR_HASH: 35, /* # */
+  CHAR_SPACE: 32, /*   */
+  CHAR_NO_BREAK_SPACE: 160, /* \u00A0 */
+  CHAR_ZERO_WIDTH_NOBREAK_SPACE: 65279, /* \u00A0 */
```

Comment:
This should be commented as `\uFEFF`, not `\u00A0`.

Reply:
@apapirovski fixed